CHANGES.rst

```diff
@@ -22,13 +22,14 @@ releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_
   with the law (:ghuser:`Eric-Sommer`)
 * Implement Art. 3 of *Familienentlastungsgesetz* on income tax tariff and child tax
   allowance becoming effective in 2020 (:ghuser:`Eric-Sommer`)
-* Implement parity in health care contributions since
-    2019 and 2020 contribution rates (:ghuser:`Eric-Sommer`)
+* Implement parity in health care contributions since 2019 and 2020 contribution rates
+  (:ghuser:`Eric-Sommer`)
 * Add *Elterngeld* calculation (:ghuser:`MaxBlesch`, :ghuser:`boryana-ilieva`)
 * Fix Soli 1991 calculation, improve Soli 1995 calculation, add 2021 Soli
   parameters and add Soli tests (:ghuser:`mjbloemer`, :ghuser:`MaxBlesch`)
 * Implement pre-2010 ruling on *Vorsorgeaufwendungen* (:ghuser:`Eric-Sommer`)
 * ``gettsim`` is released as a conda noarch package (:ghuser:`tobiasraabe`)
+* Renamed all data columns to german names (:ghuser:`amageh`, :ghuser:`MaxBlesch` )
```

Comment:
Nit-picky: Please use correct capitalisation (German); I updated it in the table with columns already

Reply:
It is German now!